.github/workflows/ci.generate.ts

```diff
@@ -7,18 +7,22 @@ import * as yaml from "https://deno.land/std@0.173.0/encoding/yaml.ts";
 // automatically via regex, so ensure that this line maintains this format.
 const cacheVersion = 59;
 
-const Runners = (() => {
-  const ubuntuRunner = "ubuntu-22.04";
-  const ubuntuXlRunner = "ubuntu-22.04-xl";
-  const windowsRunner = "windows-2022";
-  const windowsXlRunner = "windows-2022-xl";
+const ubuntuRunner = "ubuntu-22.04";
+const ubuntuXlRunner = "ubuntu-22.04-xl";
+const windowsRunner = "windows-2022";
+const windowsXlRunner = "windows-2022-xl";
+const macosX86Runner = "macos-12";
+// https://github.blog/2023-10-02-introducing-the-new-apple-silicon-powered-m1-macos-larger-runner-for-github-actions/
+const macosArmRunner = "macos-13-xlarge";
 
+const Runners = (() => {
   return {
     ubuntuXl:
       `\${{ github.repository == 'denoland/deno' && '${ubuntuXlRunner}' || '${ubuntuRunner}' }}`,
     ubuntu: ubuntuRunner,
     linux: ubuntuRunner,
-    macos: "macos-12",
+    macos: macosX86Runner,
+    macosArm: macosArmRunner,
```

Comment:
I think we only want to run this on tags? Otherwise it will get expensive fast.

Reply:
I think so -- once I get the build working I'll set skip_pr for both arm runners.

Reply:
aarch64 builds appear to be working, so I started setting skip_pr for them